### ISSUE TITLE
apertium_wikistats_bot fails to get login token

### ISSUE THREAD

**Issue by jonorthwash** [apertium/phenny#598]
```bash
$ python3 apertium_wikistats_bot.py -p kaz-kir -v 'USERNAME' 'PASSWORD' dict
/data/apertium/git/tools/phenny/apertium_wikistats_bot.py:87: SyntaxWarning: "is" with a literal. Did you mean "=="?
  for countName in sorted(fileCounts.keys(), key=lambda countName: (fileCounts[countName] and fileCounts[countName][0] is 0, countName)):
/data/apertium/git/tools/phenny/apertium_wikistats_bot.py:98: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if count is 0:
/data/apertium/git/tools/phenny/apertium_wikistats_bot.py:106: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if count is 0:
/data/apertium/git/tools/phenny/apertium_wikistats_bot.py:137: SyntaxWarning: "is" with a literal. Did you mean "=="?
  for countName in sorted(fileCounts.keys(), key=lambda countName: (fileCounts[countName] and fileCounts[countName][0] is 0, countName)):
/data/apertium/git/tools/phenny/apertium_wikistats_bot.py:169: SyntaxWarning: "is" with a literal. Did you mean "=="?
  for countName in sorted(fileCounts.keys(), key=lambda countName: (fileCounts[countName] and fileCounts[countName][0] is 0, countName)):
/data/apertium/git/tools/phenny/apertium_wikistats_bot.py:370: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(fileCounts) is 0:
DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): wiki.apertium.org:80
DEBUG:urllib3.connectionpool:http://wiki.apertium.org:80 "POST /w/api.php?action=login&format=json&lgname=USERNAME&lgpassword=PASSWORD HTTP/1.1" 301 407
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): wiki.apertium.org:443
DEBUG:urllib3.connectionpool:https://wiki.apertium.org:443 "GET /w/api.php?action=login&format=json&lgname=USERNAME&lgpassword=PASSWORD HTTP/1.1" 200 241
CRITICAL:root:Failed to login: 'login'
DEBUG:urllib3.connectionpool:http://wiki.apertium.org:80 "GET /w/api.php?action=query&format=json&prop=info&intoken=move&titles=Main+Page HTTP/1.1" 301 407
DEBUG:urllib3.connectionpool:https://wiki.apertium.org:443 "GET /w/api.php?action=query&format=json&prop=info&intoken=move&titles=Main+Page HTTP/1.1" 200 399
ERROR:root:Failed to obtain move token: 'movetoken'
DEBUG:urllib3.connectionpool:http://wiki.apertium.org:80 "GET /w/api.php?action=query&format=json&prop=info%7Crevisions&intoken=edit&titles=Main+Page HTTP/1.1" 301 419
DEBUG:urllib3.connectionpool:https://wiki.apertium.org:443 "GET /w/api.php?action=query&format=json&prop=info%7Crevisions&intoken=edit&titles=Main+Page HTTP/1.1" 200 428
DEBUG:root:edit token: +\
CRITICAL:root:Failed to obtain required token

```

**Comment by jonorthwash** [comment]
Appears to be due to this?
https://lists.wikimedia.org/pipermail/mediawiki-api-announce/2020-April/000153.html

**Comment by jonorthwash** [comment]
I think I fixed this in or around 5a9e5a76fdf4b900023213138282eecad771641c.
